src/styles/variables.css

```diff
@@ -92,4 +94,9 @@
   /* transition */
   --transition-hover: 200ms;
   --transition-appear: 500ms;
+
+  /* weights */
+  --weight-body: 400;
```

Comment:
What do you think of this comment?
https://github.com/abby-moffitt/GA-Semantic/pull/3#discussion_r1198339778

Reply:
Yeah! I see where the problem is now :/ I will update this PR accordingly